README.md

```diff
@@ -20,7 +20,7 @@ Or you can obtain a binary from https://github.com/uw-labs/strongbox/releases
 
 ## Usage
 
-1. As a one time action, install the plugin by running `strongbox git-config`.
+1. As a one time action, install the plugin by running `strongbox -config`.
```

Comment:
-config is somewhat misleading as this option configures git, not strongbox. -git-config is more descriptive.